Mobile/Android/Templates/471-reliable_uploads_actions_uploads_view.md

```diff
@@ -0,0 +1,64 @@
+|16| Folder error | 1. Tap in upload and select a file, selecting a location folder<br> 2. On server, delete this folder<br> 3. Tap on the file| The file appears on menu "Failed Uploads" with the error reason
+|17| Conflict error | 1. Tap in upload and select a file <br> 2. Upload the same file again, in the same folder| 1. The file is uploaded<br> 2. The file appears on menu "Failed Uploads" with the error reason
+|18| File error | 1. Tap to upload a file and not select it<br> 2. On server, delete the file<br> 3. Tap on the file| The file appears on menu "Failed Uploads" with the error reason
+|19| Privileges error | 1. On server, share a folder with an user without permissions to edit<br> 2. Login with the sharee<br> 3. Upload a file into the shared folder| The file appears on menu "Failed Uploads" with the error reason
+|20| Deleted account| 1. Tap to upload files<br> 2. Select several files to upload <br> 3. During the uploading, the account is removed from server<br> 4. Open the app in the uploads view with another user| The uploadings are cancelled (no file uploaded) and are shown in "Failed Uploads" section
+|21| Closed app| 1. Tap to upload files<br> 2. Select several files to upload <br> 3. During the uploading, the app is closed<br> 4. Open the app in the uploads view| The uploadings are cancelled (no file uploaded) and are shown in "Failed Uploads" section
+|**Multiaccount**||||
+|22| Upload files| 1. Login in two accouts<br> 2. Select files to upload in both accounts | The files are correctly uploaded in each account and shown in the same list of uploads view
+|**Instant Uploads**||||
+|23| Upload a pic (copy)| 1. Enable "Instant picture upload" in settings menu<br> 2. Select "copy" as behaviour<br> 3. Take a pic| The file appears in upload view in "Current" and when it finishes, in "Uploaded". Pic is correctly copied (remains in origin)
+|24| Upload a pic (move)| 1. Enable "Instant picture upload" in settings menu<br> 2. Select "move" as behaviour<br> 3. Take a pic| The file appears in upload view in "Current" and when it finishes, in "Uploaded". Pic is correctly moved (deleted in origin)
+|25| Upload a video (copy)| 1. Enable "Instant video upload" in settings menu<br> 2. Select "copy" as behaviour<br> 3. Take a video| The file appears in upload view in "Current" and when it finishes, in "Uploaded". Video is correctly copied (remains in origin)
+|26| Upload a pic (move)| 1. Enable "Instant video upload" in settings menu<br> 2. Select "move" as behaviour<br> 3. Take a video| The file appears in upload view in "Current" and when it finishes, in "Uploaded". Video is correctly moved (deleted in origin)
+|**Several Actions**||||
+|27| Upload + Share| 1. Tap to upload a file<br> 2. Select File1 to upload<br>3. While it is being uploaded, share a File2 with users/groups | File1 appears in "Uploaded" when the upload is finished. File2 is accesible for the sharees
```

Comment:
@davivel @masensio is it needed this? I mean check the uploads activity itself
I'd say that there shouldn't be any changes , so: no needed tests from 27 to 33
thx